tests/conftest.py

```diff
@@ -1,6 +1,7 @@
 import shutil
 from pathlib import Path
 
+import hyp3_sdk as sdk
```

Comment:
I don't feel very strongly about it, but I prefer not to use `as` imports unless it is common convention,

Reply:
I do this myself out of habit b/c I find the number of "hyp3"s in common commands super annoying:

```python
hyp3 = hyp3_sdk.HyP3(hyp3_sdk.HYP3_TEST)
```

I think this reads much better:

```python
hyp3 = sdk.HyP3(sdk.HYP3_TEST)
```
but in general, I agree as it's pretty brazen of us to steal the generic `sdk` namespace, so done.